v3

```diff
@@ -0,0 +1 @@
+.
```

Comment:
is this a symlink, right?

it's a nice hack .. but does this work outside of Unix at all?

Reply:
Yeap, it's a symlink.

Unfortunately, I don't have Windows to check. But I bet it doesn't work on Windows.

Anyway, the official approach is to use go 1.9.7+ or go 1.10.3+ and drop support of older versions.

And with this symlink we give Unix users with older go versions a chance to keep using it for a while.

As you can see, CI with go 1.7.x and 1.8.x passed successfully